breathecode/certificate/serializers.py

```diff
@@ -155,3 +157,11 @@ class UserSpecialtySerializer(serpy.Serializer):
     updated_at = serpy.Field()
     created_at = serpy.Field()
     issued_at = serpy.Field()
+
+    profile_academy = serpy.MethodField()
+
+    def get_profile_academy(self, obj):
+        profileAcademy = ProfileAcademy.objects.filter(user__id=obj.user.id).first()
```

Comment:
the user can be `null`, if obj.user is none return `None` and prevent do this query, also, the right `ProfileAcademy` need be filter by `academy__id` because the same user can have many ProfileAcademy, one for `Acacemy`